skyportal/handlers/api/gcn.py

```diff
@@ -171,7 +172,11 @@ def post_gcnevent_from_xml(
 
     if event is None:
         event = GcnEvent(
-            dateobs=dateobs, sent_by_id=user_id, trigger_id=trigger_id, aliases=aliases
+            dateobs=dateobs,
+            sent_by_id=user_id,
+            trigger_id=trigger_id,
+            aliases=aliases,
+            gcn_type="voevent",
```

Comment:
Quick question. Why do we need to track this on the event? I mean to me JSON/VOEVENT is purely for the notice and shouldn't matter for this event itself.

Reply:
We could have both voevent notices and JSON notices for the same event

Reply:
Ok I moved it over to the notice.